aspnetcore/mvc/razor-pages/index.md

```diff
@@ -132,13 +133,18 @@ The *MyApp/Pages/Contact.cshtml* file:
 </html>
 ```
 
-The  page has a `OnPostAsync` *handler method* which runs on `POST` requests (when a user posts the form). You can add handler methods for any HTTP verb. You most frequently use an `OnGet` handler to initialize any state a needed to show the HTML and `OnPost` to handle form submissions. The `Async` naming suffix is optional but is often used by convention. The code that's in `OnPostAsync` in the preceding example looks similar to what you would normally write in a controller. This is typical for pages. Most of the MVC primitives like model binding, validation, and action results are shared.
+The page has a `OnPostAsync` *handler method* which runs on `POST` requests (when a user posts the form). You can add handler methods for any HTTP verb. The most common handlers are:
+
+* `OnGet` to initialize state needed for the page.
+* `OnPost` to handle form submissions. 
+
+The `Async` naming suffix is optional but is often used by convention. The code that's in `OnPostAsync` in the preceding example looks similar to what you would normally write in a controller. This is typical for pages. Most of the MVC primitives like model binding, validation, and action results are shared.
 
 The basic flow of `OnPostAsync` is:
 
 1. Check for validation errors.
-1. If there are no errors, save the data and redirect -
-1. Else, show the page again with the validation errors.
+1. If there are no errors, save the data and redirect.
+1. Else if errors, show the page again with the validation message.
```

Comment:
If you have "If errors" do you need "else"?  Also, might be better for consistency to make 2 and 3 parallel: either "if errors/no errors" or "if there are errors/no errors"